jfuzzylite/src/main/java/com/fuzzylite/FuzzyLite.java

```diff
@@ -40,11 +42,14 @@
 
         @Override
         protected DecimalFormat initialValue() {
-            return new DecimalFormat("0.000");
+            NumberFormat df = NumberFormat.getNumberInstance(Locale.ROOT);
+            df.setMinimumFractionDigits(decimals);
+            df.setMaximumFractionDigits(decimals);
+            return (DecimalFormat) df;
         }
     }
 
-    private static final ThreadSafeDecimalFormat FORMATTER = new ThreadSafeDecimalFormat();
+    private static final ThreadLocal<DecimalFormat> FORMATTER = new ThreadSafeDecimalFormat();
```

Comment:
Why back to generics? I prefer the `ThreadSafeDecimalFormat` because the resulting class is FuzzyLite$ThreadSafeDecimalFormat.class, or something like that I recall.

Reply:
This was unrelated and more of a style issue IMO. I can roll it back.